### PR TITLE
[release-1.4] Don't expect the config blob to be listed in (skopeo inspect)

### PR DESCRIPTION
**Pull request by mtrmac**
This a backport of https://github.com/containers/skopeo/pull/1572 to fix CI on this branch.

### CODE REVIEW

File: systemtest/010-inspect.bats [hunk 1]
```diff
@@ -27,11 +27,19 @@ load helpers
...
     inspect_local=$output
-
-    # Each SHA-named file must be listed in the output of 'inspect'
+    run_skopeo inspect --raw dir:$workdir
+    inspect_local_raw=$output
+    config_digest=$(jq -r '.config.digest' <<<"$inspect_local_raw")
+
+    # Each SHA-named layer file (but not the config) must be listed in the output of 'inspect'.
+    # As of Skopeo 1.6, (skopeo inspect)'s output lists layer digests,
+    # but not the digest of the config blob ($config_digest), if any.
```
Comment (cevich) [inline systemtest/010-inspect.bats:36]: Non-blocking: This is the 1.4 branch, so the comment is slightly confusing.  Is it saying "Prior to 1.6, the output also included the digest of the config blob"?

**Review by cevich — APPROVED** [review]
LGTM

File: systemtest/010-inspect.bats [hunk 2]
```diff
@@ -27,11 +27,19 @@ load helpers
...
     run_skopeo inspect dir:$workdir
     inspect_local=$output
-
-    # Each SHA-named file must be listed in the output of 'inspect'
+    run_skopeo inspect --raw dir:$workdir
+    inspect_local_raw=$output
+    config_digest=$(jq -r '.config.digest' <<<"$inspect_local_raw")
+
+    # Each SHA-named layer file (but not the config) must be listed in the output of 'inspect'.
+    # As of Skopeo 1.6, (skopeo inspect)'s output lists layer digests,
```
Comment (TomSweeneyRedHat) [inline systemtest/010-inspect.bats:35]:
```suggestion
    # As of Skopeo 1.6, (skopeo inspect's) output lists layer digests,
```

Comment (TomSweeneyRedHat) [reply systemtest/010-inspect.bats:35]: Although I think I'd just dro the parens all together.

**Comment by TomSweeneyRedHat** [comment]
Doc nits for consideration, ditto 1.13 variant.  LGTM otherwise and I'd be fine with the docs as is.

File: systemtest/010-inspect.bats (hunk 1 above)
Comment (mtrmac) [reply systemtest/010-inspect.bats:36]: It is meant to say “as of all existing versions to date, up to and including 1.6”.

File: systemtest/010-inspect.bats (hunk 2 above)
Comment (mtrmac) [reply systemtest/010-inspect.bats:35]: The parens are a somewhat-weird habit to quote shell commands in a way where copy-pasting both with and without the parens is a valid shell command (unlike using `"`, or `'`, let alone typographically-correct quotes); so moving the `'s` inside doesn’t work.

The parens are an habit from a specific niche period of the Fedora IRC universe, obscure enough to be confusing, I’ll drop them.

**Comment by mtrmac** [comment]
These PRs are supposed to be backports — I’ll prepare fixes on the main branch, and add them to these PRs.

**Comment by mtrmac** [comment]
> These PRs are supposed to be backports — I’ll prepare fixes on the main branch,

https://github.com/containers/skopeo/pull/1576 .

**Comment by mtrmac** [comment]
Updated by adding #1576.

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**